### PR TITLE
Add the ability to customize timeouts for ElasticSearch

### PR DESCRIPTION
**Pull request by ghost**
**What this PR does / why we need it**:

On smaller clusters, ElasticSearch pods often have to be restarted and it takes a long time to self heal, if it happens at all. `readinessProbe` checks often fail. Add the ability to customize `readinessProbe` and `livenessProbe` for client, master and data nodes.

### CODE REVIEW

File: incubator/elasticsearch/values.yaml [hunk 1]
```diff
@@ -108,3 +121,10 @@ data:
     enabled: false
     # minAvailable: 1
     maxUnavailable: 1
+  readinessProbe:
+    initialDelaySeconds: 5
+    periodSeconds: 10
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 90
+    periodSeconds: 10
```
Comment (rekcah78) [inline incubator/elasticsearch/values.yaml]: Hello @artyomboyko, you must add a new line at the end of file for pass CI lint.

**Comment by rekcah78** [comment]
@artyomboyko you must to increase the version of chart

**Comment by artyomboyko** [comment]
@davidkarlsen 

done.

**Comment by desaintmartin** [comment]
Hello,
Could you rebase your branch? This looks interesting (I've just got a data node that got blocked for some reason but java process was kept alive, so I had to manually delete it)

**Pull request closed without merging**